### PR TITLE
Bug 1198002 - Fix broken --parameters switch for process

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@smarterclayton  @soltysh @bparees PTAL

I needed to move the printing function to process command as the function is not called from printer (it was replaced with just printing the template details?). And since we are not printing the Template but just parameters, it does not fell to the 'printer.go'

**Comment by soltysh** [comment]
LGTM

File: pkg/cmd/cli/cmd/process.go [hunk 1]
```diff
@@ -89,7 +110,7 @@ Examples:
 			// If 'parameters' flag is set it does not do processing but only print
 			// the template parameters to console for inspection.
 			if cmdutil.GetFlagBool(cmd, "parameters") == true {
-				err = printer.PrintObj(templateObj, out)
+				err = printTemplateParameters(templateObj, out)
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/process.go]: Why aren't you printing the template?

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/process.go]: Ah, you should be passing templateObj.Parameters here, and then keep printTemplateParameters in describe/printer.go (and register template parameters as its own type).

Comment (mfojtik) [inline pkg/cmd/cli/cmd/process.go]: @smarterclayton template parameters are not API objects AFAIK, I think I tried this before and the PrintObj require runtime.Object

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/process.go]: You can make PrintTemplateParameters public then.

Comment (bparees) [inline pkg/cmd/cli/cmd/process.go]: @mfojtik I don't think clayton was suggesting using printobj, just that you should pass the Parameters object, not the full template object.  you're not using any other fields of the object.

also who was calling printTemplateParameters before? I see you moved it, but no other code was changed...so was nothing calling it before?

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/process.go]: Before, get on Template (printTemplate) called printTemplateParameters.  printTemplate now displays details of a template.

----- Original Message -----

Comment (mfojtik) [inline pkg/cmd/cli/cmd/process.go]: @smarterclayton that makes sense, I will rework this, thanks for looking!

**Comment by mfojtik** [comment]
@smarterclayton @bparees moved the func into printer.go and I made it public. It now accepts list of parameters instead of template. PTAL

**Comment by mfojtik** [comment]
@csrwng FYI: https://travis-ci.org/openshift/origin/jobs/53165161#L399

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**